hr_holidays_public/models/hr_holidays.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2015 iDT LABS (http://www.@idtlabs.sl)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from dateutil import tz
+import pytz
```

Comment:
Import structure follow OCA guidelines https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror

Reply:
@hieulucky111 dateutil is not an external dependency, it's included in odoo requirements..